saskatoon/sitebase/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+class Homepage(models.Model):
+
+    pickleader_instructions = models.TextField(
+        blank=True,
+        null=True,
+        verbose_name=_("Pickleader Instructions in English"),
+        help_text=_("Steps to pick a leader"),
+        max_length=1000
+    )
```

Comment:
I believe we need to either have both english and french versions for the TextField, or have a language field, e.g.
    - volunteer_instructions
    - pickleader_instructions
    - language (EN/FR/..)
    - status (ACTIVE / DRAFT)